src/DssSpell.t.base.sol

```diff
@@ -405,13 +404,6 @@ contract DssSpellTestBase is Config, DssTest {
         );
         }
 
-        // box value in RAD
-        {
-            uint256 normalizedBox = values.cat_box * RAD;
```

Comment:
You might want to remove `cat_box` and all other cat and flip-related values from the `config.sol` as they will become unused

Reply:
I would not be doing this in the current spell but instead in the clean up PR.

Reason for this would be time concerns. We're already behind the schedule and it would take time finding & removing all the outdated functionatliy. I dont think we also want to review this along with all the checkboxes we already have in the checklist.

Reply:
See the changes.